### PR TITLE
First Commit To Creating font-awesome Addon

### PR DESCRIPTION
**Pull request by nadnoslen**

### CODE REVIEW

File: blueprints/ember-cli-font-awesome-sass/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*jshint node:true*/
+module.exports = {
+  description: '',
+
+  // locals: function(options) {
+  //   // Return custom template variables here.
+  //   return {
+  //     foo: options.entity.options.foo
+  //   };
+  // }
```
Comment (cherylcourt) [inline blueprints/ember-cli-font-awesome-sass/index.js]: Did you mean to leave this commented out code in this file?

**Review by cherylcourt — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+applications that are compiling their styles using SASS.  __If your Ember
+application is using pure CSS or LESS, than this addon is not for you.__
+
+This addon will make sure that the latest version of the
+[`ember-cli-sass` Ember Addon](https://github.com/aexmachina/ember-cli-sass) is
+install.  In addition, the latest version of the
+[`font-awesome` library](http://fontawesome.io/) from bower will be installed
+in order to copy over the fonts and consume the SASS source.
+
+## Installation
```
Comment (nadnoslen) [inline README.md:16]: Added some installation documentation.

File: app/styles/_ember-cli-font-awesome-sass.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * OVERRIDE: if you want to point to fonts installed elsewhere, change this
+ * variable before you `@import "ember-cli-font-awesome-sass"`.
+ */
+$fa-font-path: '/assets/font-awesome/fonts';
+
+/**
+ * The `ember-cli-font-awesome-sass` addon installs font-awesome via bower
+ * and then compiles font-awesome using their SASS source.
+ */
+@import "../../bower_components/font-awesome/scss/font-awesome";
```
Comment (nadnoslen) [inline app/styles/_ember-cli-font-awesome-sass.scss:13]: This file is the one that someone's app will import for compilation.  font-awesome will then be available.

File: blueprints/ember-cli-font-awesome-sass/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+/*jshint node:true*/
+module.exports = {
+  description: '',
+
+  afterInstall: function(options) {
+    return Ember.RSVP.all([
+      this.addAddonToProject('ember-cli-sass'),
+      this.addBowerPackageToProject('font-awesome')
+    ]);
```
Comment (nadnoslen) [inline blueprints/ember-cli-font-awesome-sass/index.js:9]: I need to see if this technique actually works.  Will npm link one of my apps.

File: index.js [hunk 5]
```diff
@@ -2,5 +2,20 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-cli-font-awesome-sass'
+  name: 'ember-cli-font-awesome-sass',
+  included: function (app) {
+    this._super.included(app);
+    app.import(app.bowerDirectory + "/font-awesome/fonts/FontAwesome.otf",
+      { destDir: "assets/font-awesome/fonts" });
```
Comment (nadnoslen) [inline index.js:9]: I created issue #2 in order to consider improving destDir to be configured by an environment variable.

Comment (cherylcourt) [reply index.js:9]: I saw that. 👍

**Review by cherylcourt — APPROVED** [review]
Neat.

**Comment by cherylcourt** [comment]
Are you still working out of this branch or should it be deleted?

**Pull request merged**